### ISSUE TITLE
Fix information concerning deal polling on the about page

### ISSUE THREAD

**Issue by alanshaw** [web3-storage/web3.storage#1042]
> Once the deals are active, the Web3.Storage client polls to ensure that the relevant sectors are still available. In the event of an early termination (or a miner going offline), the Web3.Storage client will automatically add the relevant deals into the queue of upcoming deals to ensure at all times there are always a minimum of 5 copies of data being stored with Filecoin Storage Providers.
> https://web3.storage/about/

This is incorrect. The _client_ does not do this. Dagcargo maybe does something _like_ this.

cc @ribasushi

**Comment by vasco-santos** [comment]
@dchoi27 synced with @ribasushi and at least now we don't support it, even though dagcargo aspires to support it.

In the meantime we should rephrase this to not include it. And also it is a layer underneath the web3storage client.

What is you suggestion here? Focus more on Filecoin capabilitiy of proof of storage until we have a technical solution in place?

**Comment by olizilla** [comment]
@dchoi27 should we just remove this text for now, until we make it true?

**Comment by dchoi27** [comment]
Agreed we should get rid of the blurb for now. opened #1407 to address this - could use help getting rid of the blurb!
